backend/redis/utils.go

```diff
@@ -519,6 +523,14 @@ func (r *Client) ResetServiceErrorCount(ctx context.Context, projectId int) (int
 	return r.redisClient.Del(ctx, serviceKey).Result()
 }
 
+func (r *Client) SetGitHubFileError(ctx context.Context, gitHubRepo string, version string, fileName string) error {
+	return r.setFlag(ctx, GitHubFileErrorKey(gitHubRepo, version, fileName), true, 1*time.Hour)
```

Comment:
How do we feel about one hour? We could do more, my only concern is increasing our redis cache size. Are there better ways to cache (i.e. store an empty file in S3)?

Reply:
nah, redis is great for this - just a key and a boolean so it should be very efficient (i believe the key is hashed to an efficient index in redis)

hour sounds good for sure